packages/cli/src/lib/api-helpers.ts

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash'
+import { SmartThingsClient } from '@smartthings/core-sdk'
+
+
+export interface LocationItem {
+	locationId?: string
+	location?: string
+}
+
+export interface LocationRoomItem extends LocationItem {
+	roomId?: string
+	room?: string
+}
+
+export interface RoomItem {
+	locationId?: string
+	roomId?: string
+	room?: string
+}
+
+class IdNameMap {
+	[name: string]: string
+}
+
+class NestedIdNameMap {
+	[name: string]: {[name: string]: string}
+}
+
+export async function addLocations(client: SmartThingsClient, list: LocationItem[]): Promise<void> {
+	const locations = await client.locations.list()
+	const names: {[name: string]: string} = locations.reduce((map, obj) => {
+		map[obj.locationId] = obj.name
+		return map
+	}, new IdNameMap())
+
+	for (const item of list) {
+		if (item.locationId) {
+			item.location = names[item.locationId]
+		} else {
+			item.location = ''
+		}
+	}
+}
+
+export async function addLocationsAndRooms(client: SmartThingsClient, list: LocationRoomItem[]): Promise<void> {
+	const locations = await client.locations.list()
+	const locationMap: {[name: string]: string} = locations.reduce((map, obj) => {
+		map[obj.locationId] = obj.name
+		return map
+	}, new IdNameMap())
+
+	// eslint-disable-next-line no-console
+	console.log(`LOCATIONS = ${JSON.stringify(locationMap)}`)
```

Comment:
Guessing you didn't mean to leave this in but if you did, we should use the logger.